administrate-field-nested_has_many.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.files = `git ls-files`.split("\n")
   gem.test_files = `git ls-files -- {test,spec,features}/*`.split("\n")
 
-  gem.add_dependency "administrate", ">= 0.15", "< 0.20.0"
+  gem.add_dependency "administrate", ">= 0.19", "< 1"
```

Comment:
After pinning in #68, this effectively brings up the minimum (which I'd kind of like to avoid, but hmm) and restores the original constraint.

I intend to pull this out of this PR before we merge it (if not just so there's a separate line in the CHANGELOG).